dev/dev/devDeployment.yaml

```diff
@@ -61,7 +61,8 @@ oidc-proxy:
   env:
     PROXY_URL: http://dev-jade-datarepo-ui.dev:8080/
     PROXY_URL2: http://dev-jade-datarepo-api.dev:8080/api
-    PROXY_URL3: http://dev-jade-datarepo-api.dev:8080/register
+    PROXY_URL3: http://se-jade-datarepo-api.se:8080/ga4gh
```

Comment:
Oh - I should have caught this when I was reading it before - but this should be "dev", not "se"